### PR TITLE
OTEL Collector docs edits

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Changes carried over from **https://github.com/SumoLogic/sumologic-documentation/pull/1872**. To address:

- [Who is the target audience for Get Started > Quickstart](https://github.com/SumoLogic/sumologic-documentation/pull/1872#discussion_r1149842169)? Need to distinguish between new trial users and existing users that are setting up a new source. The OpenTelemetry data collection and onboarding has a slightly different workflow than what is mentioned in this page. We should modify it to match our new workflow.
- [For Send Data/Index: Not sure if we should have the "JSON Configuration" tile](https://github.com/SumoLogic/sumologic-documentation/pull/1872#discussion_r1149855985) since this talks about how to configure the source on the Installed/hosted collector. Should be a subtopic within Installed/Hosted Collector. We don't have a section here that talks about configuring the source on OpenTelemetry Collector.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md [hunk 1]
```diff
@@ -16,14 +25,24 @@ All Sumo Logic preconfigured components are stored in `sumologic.yaml` file. Thi
...
+
+Use Custom configuration to customize the collection of your logs, metrics and traces in Sumo Logic. Learn more about configuration [here](https://opentelemetry.io/docs/collector/configuration).
+
+<!--link to other sources from OpenTelemetry that will help users understand more about configuration.-->
 
 There are few processors provided in `sumologic.yaml` which are intended to be used in every pipeline.
 
 * **Memory limiter processor**. It is used to prevent out-of-memory situations on the collector. It should be always first on the processor's list. For more information, refer to the [OpenTelemetry documentation](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/memorylimiterprocessor#memory-limiter-processor).
 * **Batch processor**. It accepts spans, metrics, or logs and places them into batches. Batching helps better compress the data and reduce the number of outgoing connections required to transmit the data. See [Using batch processor to batch data](#using-batch-processor-to-batch-data) for more information.
+<!--There is another processor called ResourceDetection/system. Should that also be used?-->
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md:45]: @sumoanema @arunpatyal @ajaiswals , can you confirm?

File: docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md [hunk 2]
```diff
@@ -16,14 +25,24 @@ All Sumo Logic preconfigured components are stored in `sumologic.yaml` file. Thi
 * It is recommended to maintain the configuration of all reusable components in `conf.d/common.yaml`.
 * Specific configurations should be grouped by pipelines in meaningfully named files in the `conf.d` directory.
 
+The `conf.d` folder has been created by Sumo Logic, and should contain a separate configuration file for a specific application component for example MySQL or Nginx.
+
 For example, a file named `conf.d/mysql.yaml` can contain the MySQL receiver along with any processors that are intended to modify the collected data before sending it to Sumo Logic.
 
-## Custom Pipeline
+The **App Catalog** provides a mechanism to create these configuration files using a simple UI form input. Learn More
+<!-- Learn More should be a link to App Catalog page that explains how the Souce Configuration section works.-->
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/additional-configurations-reference.md:33]:
```suggestion
<!-- Learn More should be a link to App Catalog page that explains how the Source Configuration section works.-->
```

**Review by angadrandhawa1 — APPROVED** [review]

File: docs/get-started/quickstart.md [hunk 3]
```diff
@@ -40,7 +40,27 @@ The journey of 10,000 logs begins with a single collector. Your data analytics j
...
 ### Set up Collector
 
-From the Sumo Home page, go to **Manage Data** > **Collection** > **Collection** tab, and select **Add Collector**. Next, choose from a local [Installed Collector](/docs/send-data/installed-collectors), web-[Hosted Collector](/docs/send-data/hosted-collectors), or our [OpenTelemetry Collector](/docs/send-data/opentelemetry-collector). Next, you'll choose the data sources that will provide the most value for you.
+After you've logged into Sumo Logic, choose a collector.
+
+#### OpenTelemetry Collector
+
+[OpenTelemetry Collector](/docs/send-data/opentelemetry-collector)
+
+1. Go to **App Catalog** and click on the app you'd like to install.
```
Comment (stacykor) [inline docs/get-started/quickstart.md:49]:
```suggestion
1. Go to **App Catalog** and click the app you'd like to install.
```

**Review by stacykor — CHANGES_REQUESTED** [review]
small edit

**Pull request merged**